examples/reference_environments/postgres/jupyter.Dockerfile

```diff
@@ -5,3 +5,7 @@ WORKDIR /gx
 COPY ./postgres_example.ipynb ./
 
 RUN pip install great_expectations[postgresql]
+
+# Use this line to install GX from the develop branch,
+# or replace develop with the branch name you want to install from:
+RUN pip install git+https://github.com/great-expectations/great_expectations.git@develop
```

Comment:
I thought it might be more likely that during debugging we would be using develop or another branch rather than the latest release so I included this, and maybe we consider adding this to other ref envs (with instructions to rebuild on change).